pennylane_lightning/lightning_base.py

```diff
@@ -119,12 +135,30 @@ def capabilities(cls):
         # To be able to validate the adjoint method [_validate_adjoint_method(device)],
         #  the qnode requires the definition of:
         # ["_apply_operation", "_apply_unitary", "adjoint_jacobian"]
+        # pylint: disable=no-method-argument
         def _apply_operation():
             pass
 
+        # pylint: disable=no-method-argument, missing-function-docstring
         def _apply_unitary():
             pass
 
+        # pylint: disable=no-method-argument, missing-function-docstring, assignment-from-no-return
```

Comment:
I wonder if there is a better way than these

Reply:
Do you mean the `# pylint: disable` or the trivial methods?

Reply:
The multiple disable statements. We define a lot of them here. Maybe this is the only way

Reply:
I'll try to go easier on the disable.